experimental/docker-stacks-and-bundles.md

```diff
@@ -52,7 +52,7 @@ Options:
 Let's deploy the stack created before:
 
 ```bash
-# docker deploy vossibility-stack
+# docker deploy vossibility-stack  vossibility
```

Comment:
I don't think this change is correct, because `vossibility-stack` is already the name that's been given to the stack (see the output below as well).

It looks like this page is missing some changes, i.e.; `--file` has been renamed `--bundle-file` (for `.dab` and `--compose-file`), and is a required flag.